docs/configuring.md

```diff
@@ -0,0 +1,254 @@
+|**MAXPROCSYS**     | 900        |
+|**MAXPROCUSER**    |	512        |
+|**MAXUIDS**	      | 500        |
+|**MAXTHREADS**     |	10000      |
+|**MAXTHREADTASKS**	| 5000       |
+|**MAXASSIZE**	    | 2147483647 |
+|**MAXCPUTIME**	    | 2147483647 |
+|**MAXMMAPAREA** 	  | 40960      |
+|**IPCSEMNIDS**	    | 500        |
+|**IPCSEMNSEMS**	  | 1000       |
+
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** The number of threads that can be created by a Java process is limited by the lower of the two values for **MAXTHREADS** and **MAXTHREADSTASKS**.
+
+You can change these settings dynamically without re-IPLing the system. For example, to set **MACPROCUSER** to 256, run `SETOMVS MAXPROCUSER=256`
+
+z/OS uses region sizes to determine the amount of storage available to running programs. For a Java runtime environment, the  region size must be sufficiently large to avoid storage related error messages or abends. Rather than restricting region size, allow the VM to use what it needs. Region size can be affected by one of the following parameters: **JCL REGION**, **BPXPRMxx MAXASSIZE**, the RACF OMVS segment **ASSIZEMAX**, or IEFUSI (Step initiation exit).
+
+
+### Configuring Language Environment runtime options
+
+Language Environment&reg; runtime options affect performance and storage usage. These options can be optimized for your application.
+
+Runtime options are typically embedded in programs by using **#pragma runopts** settings. In many cases, these options provide suitable default values that are known to produce good performance results. However, these options can be overridden to tune the runtime environment of your application.
+
+On 64-bit z/OS systems, the following runtime options affect Java applications:
+
+- **HEAP64**: Controls allocation of the user heap. A suggested starting point for an override is **HEAP64(512M,4M,KEEP,16M,4M,KEEP,0K,0K,FREE)**.
+- **HEAPPOOLS64**: Used to manage Java heap storage above the 2 G bar. The Java USS launcher sets **HEAPPOOLS64(ALIGN)** for more optimal management of multi-threaded applications. Other Java launchers might have different settings. Before you set an override for **HEAPPOOLS64**, use **RPTOPTS(ON)** or **RPTSTG(ON)** and check the result of **#pragma runopts**. Check this result because the host product might have already set cell sizes and numbers that are known to produce good performance.
+- **STACK64**: Controls the allocation and management of stack storage. A suggested default is **STACK64(1M,1M,128M)**.
+- **THREADSTACK64**: Controls the allocation of thread-level stack storage for both the upward and downward-growing stack. A suggested default is **THREADSTACK64(OFF,1M,1M,128M)**.
+
+A suitable **MEMLIMIT** value is also required. The OpenJ9 VM requirement is the sum of the following amounts:
+
+- **HEAP64** (minimum 512 M), which should include **HEAPPOOLS64**.
+- **STACK64** (3 MB multiplied by the expected number of concurrent threads)
```

Comment:
Similar to above:
- User Stack Storage (3MB multiplied by the expected number of concurrent threads), as controlled by **STACK64** settings.